Workbooks/WorkbooksMetadata.json

```diff
@@ -10,7 +10,8 @@
     "title": "Azure Activity",
     "templateRelativePath": "AzureActivity.json",
     "subtitle": "",
-    "provider": "Microsoft"
+    "provider": "Microsoft",
+    "featureFlag": "abc"
```

Comment:
What does the feature flag abc mean?

Reply:
I removed it for now, but its just for testing..